messaging/src/test/java/org/axonframework/common/SimpleContextTest.java

```diff
@@ -16,6 +16,8 @@
 
 package org.axonframework.common;
 
+import org.axonframework.messaging.SimpleContext;
```

Comment:
Looks like the test class needs to be moved.

Reply:
Sharp! Moved the class.